[PL-41689]: Add documentation for absolute session timeout (#4890)
harness/developer-hub: docs/platform/authentication/authentication-overview.md

   You can set this to a minimum of 30 minutes and a maximum of 4320 minutes (3 days). The field automatically converts the minutes you enter to higher units of time, and displays the result under the field. For example, if you enter 1440, the UI shows **1 day** below the field.

>    ![](./static/SessionTimeout.png)

Reviewer: Old image should be updated with new image that consists of both Session Inactivity Timeout and Absolute Session Timeout

Reviewer 2: updated with latest commit

### Set absolute session timeout

Harness logs a user out of their account after a session timeout irrespective of any activity.

> To configure your account's session inactivity timeout, do the following:

Reviewer: ```suggestion
To configure your account's Absolute Session Timeout, do the following:
```

1. In your Harness account, select **Account Settings**.

2. Select **Authentication**.

> 3. In **Absolute Session Timeout (in minutes)**, enter the time in minutes to set the absolute session time out.

Reviewer: ```suggestion
3. In **Absolute Session Timeout (in minutes)**, enter the time in minutes to set the Absolute Session Timeout.
```

>    The default absolute session timeout is 0, that means absolute session timeout is not set.

Reviewer: ```suggestion
   The default absolute session timeout is 0—which means that Absolute Session Timeout is not set.
```

   You can set this to a maximum of 4320 minutes (3 days). The field automatically converts the minutes you enter to higher units of time, and displays the result under the field. For example, if you enter 1440, the UI shows **1 day** below the field.

> In case both inactive session timeout and absolute session timeout is set, the condition first met wlll be honoured.

Reviewer: ```suggestion
In case both Session Inactivity Timeout and Absolute Session Timeout is set, the condition first met will be honoured.
```
